hash/m20.xml

```diff
@@ -159,137 +199,150 @@ license:CC0-1.0
 		</part>
 	</software>
 
-	<software name="adm5" supported="no">
+<!-- Programs, generally boot with PCOS41a unless stated otherwise, fix track0 first -->
+
+	<software name="adm5">
+	<!-- "This is a standard 360k MS-DOS disk image, 512 bytes/sector, 9 sectors, 2 sides, 40 tracks." -->
+	<!-- Use with BIOS 2.0 and the 8086 APB - boot with Msdos in drive0 -->
 		<description>ADM5 Terminal Emulator for MS-DOS</description>
 		<year>19??</year>
 		<publisher>&lt;unknown&gt;</publisher>
 
 		<part name="flop1" interface="floppy_5_25">
-			<dataarea name="flop" size="33619">
+			<dataarea name="flop" size="368640">
 				<rom name="adm5.imd" size="33619" crc="f7abfeee" sha1="ee432dc896f9b034c1f7ff3fac67cf5b8e003ff5"/>
 			</dataarea>
 		</part>
-	</software>
-
-	<software name="adm51" supported="no">
-		<description>ADM51 Terminal Emulator for MS-DOS</description>
-		<year>19??</year>
-		<publisher>&lt;unknown&gt;</publisher>
-
-		<part name="flop1" interface="floppy_5_25">
-			<dataarea name="flop" size="8192">
-				<rom name="adm51.imd" size="8192" crc="8916ea22" sha1="99fec256ae75c6569dff5bb8005f0e7bc301f4d8"/>
+		<part name="flop2" interface="floppy_5_25"><!-- Missing MFM track0 -->
+			<dataarea name="flop" size="368640">
+				<rom name="adm51.imd" size="8192" crc="8916ea22" sha1="99fec256ae75c6569dff5bb8005f0e7bc301f4d8" status="baddump"/>
 			</dataarea>
 		</part>
 	</software>
 
-<!-- Programs -->
-
-	<software name="asm" supported="no"> <!-- Invalid in M20 -->
+	<software name="asm" supported="partial"><!-- Missing FM track0 -->
 		<description>Olivetti Z8000 Assembler 2.1</description>
 		<year>19??</year>
 		<publisher>Olivetti</publisher>
 
 		<part name="flop1" interface="floppy_5_25">
 			<dataarea name="flop" size="286720">
-				<rom name="asm_r2.1.a.img" size="286720" crc="90b0fb9e" sha1="b250ff32d40293da446ce96b513b0000fb7814d1"/>
+				<rom name="asm_r2.1.a.img" size="286720" crc="90b0fb9e" sha1="b250ff32d40293da446ce96b513b0000fb7814d1" status="baddump"/>
 			</dataarea>
 		</part>
 	</software>
 
-	<software name="dietasm">
+	<software name="dietasm" supported="no"><!-- Missing FM track0, boot with -ramsize 512k -->
```

Comment:
For software that requires more RAM than the default configuration, it’s better to add that in a `usage` `info` element so it’s visible in the UI, etc.